docs/ref/contrib/admin/index.txt

```diff
@@ -1309,13 +1309,19 @@ subclass::
         WHERE (first_name ILIKE '%john%' OR last_name ILIKE '%john%')
         AND (first_name ILIKE '%lennon%' OR last_name ILIKE '%lennon%')
 
+    The search query can contain quoted phrases with spaces. For example, if a
+    user searches for ``"john winston"`` or ``'john winston'``, Django will do
+    the equivalent of this SQL ``WHERE`` clause:
+
+    .. code-block:: sql
+
+        WHERE (first_name ILIKE '%john winston%' OR last_name ILIKE '%john winston%')
+
     If you don't want to use ``icontains`` as the lookup, you can use any
     lookup by appending it the field. For example, you could use :lookup:`exact`
     by setting ``search_fields`` to ``['first_name__exact']``.
 
-    Beware that because query terms are split and ANDed as described earlier,
-    searching with :lookup:`exact` only works with a single search word since
-    two or more words can't all be an exact match unless all words are the same.
+    Beware that because query terms are split and ANDed as described earlier.
```

Comment:
Because what? This sentence is no longer complete. Can it just be dropped entirely? 
`only works with a single search word` is no longer true...

Reply:
Agreed, I think we can the entire "beware". I will do this later today.